app/main.dev.ts

```diff
@@ -125,7 +131,9 @@ const createWindow = async () => {
           disableBlinkFeatures: 'Auxclick',
           enableRemoteModule: true,
           nodeIntegration: false,
+          nodeIntegrationInWorker: false,
           preload: path.join(__dirname, 'dist/renderer.prod.js'),
+          sandbox: true,
```

Comment:
Are you able to run `sandbox: true` with `yarn build` or the package version in Linux? I get a black screen on a Mac.

Reply:
I'll try this out; I was using `yarn dev`.

Reply:
I can confirm `yarn dev` works (tested it again, to be sure) but the package built with `yarn package --linux tar.gz` failed. On further study, removing the option is wisest; done. I will include testing the built app (for Linux at least) in future code changes.